src/app/shared/components/file-preview/file-preview.component.html

```diff
@@ -23,8 +23,13 @@
     </div>
 
     <div *ngSwitchDefault>
-        <div class="default-preview preview">
-            <icon class="node-icon">insert_drive_file</icon>
+        <div class="default-preview">
+            <div class="file-icon">
+                <icon>insert_drive_file</icon>
+            </div>
+            <div class="mime-wrapper">
+                <div class="mime-type">{{ mediaType }}</div>
```

Comment:
Use a more descriptive label, transform common mime types to more human-readable labels.